topics/single-cell/tutorials/GO-enrichment/tutorial.md

```diff
@@ -33,8 +34,8 @@ key_points:
 - GO enrichment analysis is used to discover new insights about how cells work, which can lead to better understanding of biological processes and diseases.
   
```

Comment:
If you like can use our fancy new contributions syntax to more accurately record contributions to a training material, declaring which people should be recorded for authorship or just editing.

```
contributions:
  authorship:
    - shiltemann
  editing:
    - bebatut
    - hexylena
  funding:
    - carpentries
  testing:
    - userX
  ux:
    - userY
  infrastructure:
    - userZ
```

see the [full gtn tutorial schema documentation](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/schemas/tutorial.html) for details on the individual roles.